browser/ntp_background_images/ntp_background_pref.cc

```diff
@@ -0,0 +1,122 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ntp_background_images/ntp_background_pref.h"
+
+#include <utility>
+
+#include "base/notreached.h"
+#include "brave/components/constants/pref_names.h"
+#include "components/pref_registry/pref_registry_syncable.h"
+#include "components/prefs/pref_service.h"
+
+namespace {
+
+constexpr char kTypeKey[] = "type";
+constexpr char kRandomKey[] = "random";
+constexpr char kSelectedItemeKey[] = "selected_value";
```

Comment:
"kSelectedItem**e**Key" typo?
and the value is "selected_value", not "selected_item". expected?

Reply:
Oops. "selected_value" is right one. Renamed other places.